lms/djangoapps/mobile_api/models.py

```diff
@@ -25,3 +26,63 @@ def get_video_profiles(cls):
         Get the list of profiles in priority order when requesting from VAL
         """
         return [profile.strip() for profile in cls.current().video_profiles.split(",") if profile]
+
+
+class MobileAppVersionConfig(ConfigurationModel):  # pylint: disable=model-missing-unicode
+    """
+    Configuration for mobile app versions available.
+    """
+    IOS = "ios"
+    ANDROID = "android"
+    MOBILE_PLATFORM = (
+        (IOS, "IOS"),
+        (ANDROID, "ANDROID"),
+
+    )
+    KEY_FIELDS = ('platform', )  # mobile platform is unique
+    platform = CharField(
+        max_length=50,
+        choices=MOBILE_PLATFORM,
+        blank=False,
+        help_text="mobile device platform")
+    latest_version = TextField(blank=False, help_text="Latest available version for app in X.X.X format")
+    min_supported_version = TextField(blank=False, help_text="Min supported version for app in X.X.X format")
+    next_min_supported_version = TextField(
+        blank=False,
+        help_text="Next supported version for app in X.X.X format that a user must upgrade to before deadline"
+    )
+    update_required_date = DateTimeField(verbose_name="Upgrade Deadline")
+
+    def parsed_version(self, version):
+        """ Converts string X.X.X.Y to tuple (X, X, X, Y) and return (X, X, X) """
+        return tuple(version.split("."))[:3]
+
+    def is_outdated_version(self, user_app_version):
```

Comment:
Just `version` instead of `user_app_version` will reduce a lot of noise in these methods. These methods do not care what does the version belong to.